Automattic/wp-calypso
# Add key metrics component

Closes https://github.com/Automattic/dotcom-forge/issues/7300

## Proposed Changes

* Add a component for showing a single metric
* Add positive/negative header props, and control these with the score

## Testing Instructions



* Go to `/site-profiler` and enter a site URL
* Check that the basic metrics are shown similarly to the figma designs

| Device | Figma | Calypso |
|--------|--------|--------|
| Desktop | ![CleanShot 2024-05-23 at 11 26 48@2x](https://github.com/Automattic/wp-calypso/assets/11555574/ab297b6a-88e2-4ab2-8629-1c84f8a74bfe) | ![CleanShot 2024-05-23 at 11 27 10@2x](https://github.com/Automattic/wp-calypso/assets/11555574/0c17649a-0304-49b7-8e33-b32639d5e03e) |
| Mobile | ![CleanShot 2024-05-23 at 11 27 31@2x](https://github.com/Automattic/wp-calypso/assets/11555574/5873a385-f0f6-4e9c-9ff7-6421d1793bea) | ![CleanShot 2024-05-23 at 11 29 30@2x](https://github.com/Automattic/wp-calypso/assets/11555574/10227e61-b06c-41de-94c0-2909f2a7c70c) | 

### Known issues

- The site build doesn't match the figma designs in a pixel-perfect way. I found that there are fragment distances in Figma, or stylelint prevented me from using a specific pixel value. I tried to make the page as similar to the figma styles as possible, but it still may need tweaking.
- We may also need to add more breakpoints. Currently the design looks a bit off between 500-900 width.
- There is no design for the `needs-improvement` score, for now I left it with a grey color

## Pre-merge Checklist
